contrib/subrepo-cheri-libunwind/include/__libunwind_config.h

```diff
@@ -66,8 +66,8 @@
 #   ifdef __SEH__
 #    error "aarch64 SEH not supported yet"
 #   endif
-#   define _LIBUNWIND_CONTEXT_SIZE 102
-#   define _LIBUNWIND_CURSOR_SIZE 126
+#   define _LIBUNWIND_CONTEXT_SIZE 100
+#   define _LIBUNWIND_CURSOR_SIZE 124
 #   define _LIBUNWIND_HIGHEST_DWARF_REGISTER _LIBUNWIND_HIGHEST_DWARF_REGISTER_ARM64_CHERI
```

Comment:
`_LIBUNWIND_HIGHEST_DWARF_REGISTER_ARM64_CHERI` needs to be changed

Reply:
It is '229' which is the register number of C31.  I think that means it is correct?

Reply:
Oh so that was already correct then? Hm.

Reply:
Yes, I'm not sure if I had made that correct and just had not realized I should have used the dwarf numbers for C0-C31, or if it correct from Brett's changes.